pkg/kademlia/queue.go

```diff
@@ -4,152 +4,125 @@
 package kademlia
 
 import (
-	"container/heap"
-	"math/big"
+	"math/bits"
+	"sort"
 	"sync"
 
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/storj"
 )
 
-// XorQueue is a priority queue where the priority is key XOR distance
-type XorQueue struct {
+// Queue is a limited priority queue for nodes with xor distance
+type Queue struct {
```

Comment:
I feel like the name "Queue" alone will cause people to think this is a simple Queue implementation, without digging into the fact that it isn't.  Really, as its only used by peer_discovery, it should probably just be internal 'queue'

Reply:
renamed to `discoveryQueue`